### PR TITLE
Add multi imports and import tests

### LINKED ISSUE #56: Consider exporting a collected object
In the same way that we can use:

```javascript
import nameOfMacro from 'ember-awesome-macros/name-of-macro';
// or
import { nameOfMacro } from 'ember-awesome-macros';
```
 in ember-awesome-macros, do you think it would be possible to support:

```javascript
import { raw } from 'ember-macro-helpers'
```

in ember-macro-helpers?

### PR DESCRIPTION
**Pull request by mriska**
Closes #56

### CODE REVIEW

**Comment by mriska** [comment]
@kellyselden, I made all the functions that were not prefixed with `-` multi-importable, even if some of them are not documented in the readme. Would you like me to restrict it to  just the ones that are documented or is this ok?

**Comment by kellyselden** [comment]
Good question. Let's start with just the one's public in the readme.

**Comment by mriska** [comment]
@kellyselden Alright, now the PR should only contain the functions mentioned in the README.

File: tests/helpers/expect-imports.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const exclude = [
```
Comment (kellyselden) [inline tests/helpers/expect-imports.js]: this can use ember-macro-test-helpers/expect-imports now

Comment (mriska) [reply tests/helpers/expect-imports.js]: @kellyselden updated PR to use `expect-imports` from `ember-macro-test-helpers`.

**Comment by kellyselden** [comment]
Thank you sir!

**Pull request merged**